README.md

```diff
@@ -0,0 +1,58 @@
+# Example Pipeline
+
+This repo includes an example pangeo-forge pipeline using the NOAA SST dataset.
+
+## Run the example locally using conda and python
+
+Pre-requisites:
+* [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/)
+
```

Comment:
Do they need `prefect` installed? I know that this section is for local running but [this](https://github.com/pangeo-forge/example-pipeline/pull/3/files#diff-c1ba9cb202886813ca3e7aa5645ec022ebf0588625b8411e5ed004e81e1b707aR72) will require `prefect` if the user passes that argument